soh/soh/SohGui/ResolutionEditor.cpp

```diff
@@ -79,7 +79,7 @@ static float aspectRatioX;
 static float aspectRatioY;
 static int32_t verticalPixelCount;
 // Additional settings
-static bool showHorizontalResField;
+static bool showHorizontalResField = false;
```

Comment:
```suggestion
static bool showHorizontalResField;
```
static variables are always zero initialized

Reply:
Good to know. Updated.